code/src/App.js

```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { HappyThoughts } from './Components/HappyThoughts'
+import { HappyForm } from './Components/HappyForm'
+
+const url = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [postedMessage, setPostedMessage] = useState("")
```

Comment:
You are using "  " everywhere in your code except for in the imports when you use ' '. Maybe you can change the import ' ' to " " to keep the consistency through out the code? :)

Reply:
Yes, you are right, it was a mistake!(: